src/gurobi_optimods/min_cost_flow.py

```diff
@@ -61,14 +61,21 @@ def min_cost_flow_pandas(
     with create_env() as env, gp.Model(env=env) as model:
         model.ModelSense = GRB.MINIMIZE
 
+        source_label, target_label = arc_data.index.names
+
+        use_multigraph = False
+        # This is a workaround for duplicate entries being disallowed in gurobipy_pandas
```

Comment:
Since this is just internal, I'd suggest just resetting the index in all cases. Distinguishing the multigraph case doesn't add much here.